src/presentational-components/shared/loader-placeholders.js

```diff
@@ -77,22 +78,24 @@ const SkeletonContainer = styled.div`
 `;
 
 export const CardLoader = ({ items }) => (
-  <Grid gutter="md">
-    <GridItem sm={12} className="pf-u-p-md">
-      <Gallery gutter="md">
-        {[...Array(items)].map((_item, index) => (
-          <GalleryItem key={index}>
-            <Card style={{ height: 350 }}>
-              <CardBody>
-                <Skeleton height={70} width="85%" className="pf-u-mb-lg" />
-                <Skeleton height={5} width="90%" className="pf-u-mb-sm" />
-                <Skeleton height={5} width="100%" className="pf-u-mb-sm" />
-                <Skeleton height={5} width="76%" className="pf-u-mb-sm" />
-              </CardBody>
-            </Card>
-          </GalleryItem>
-        ))}
-      </Gallery>
+  <Grid>
```

Comment:
Does this not need the `hasGutter` because the old one had `gutter="md"`?

Reply:
it does!